spinalcordtoolbox/deepseg/models.py

```diff
@@ -200,8 +200,14 @@
 }
 
 
-# List of task. The convention for task names is: action_(animal)_region_(contrast)
-# Regions could be: sc, gm, lesion, tumor
+# List of tasks. The convention for task names is: action_(animal)_region_(contrast)
+#   Regions could be: sc, gm, lesion, tumor
+#   If the animal is human, omit it from the task label.
+# Some optional tags to help with groupin and sorting during help display are as follows:
+#   Group: The sub-header the tasks should be displayed under. If not specified, the model is placed under "OTHER"
+#   Priority: Higher priority tasks are displayed higher within their own group, and vice versa. Default priority is 0,
+#     and ties are sorted alphabetically.
```

Comment:
This `priority` idea is really smart! I love that it works like `nice` et al. in that we can have multiple high priority models while still fussing with the order (e.g. `100`, `99`, etc.).

It also feels a lot more intuitive than, say, a hardcoded `order` parameter, since we're indicating high/low priority intent.